ServerCore/Pages/Rooms/AddRooms.cshtml.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Rooms
+{
+    [Authorize(Policy = "IsEventAdmin")]
+    public class AddRoomsModel : EventSpecificPageModel
+    {
+        public AddRoomsModel(PuzzleServerContext serverContext, UserManager<IdentityUser> manager) : base(serverContext, manager)
+        {
+        }
+
+        public void OnGet()
+        {
+        }
+
+        public async Task<IActionResult> OnPost(string inputRooms, string group) 
+        {
+            string[] rooms = inputRooms.Split("\r\n");
```

Comment:
It's possible this will have different line breaks on Mac and mobile. Not sure if there's a great way to deal with that

Reply:
It looks like the bulk response creation uses a StringReader.ReadLine(), which checks for all EOL possibilities.  I highly recommend doing it that way.

Reply:
Looks like Environment.NewLine should handle it without requiring me to rewrite the logic.